src/celeritas/field/detail/FieldUtils.hh

```diff
@@ -81,10 +81,10 @@ inline CELER_FUNCTION Chord make_chord(Real3 const& src, Real3 const& dst)
      return ipow<2>(distance(pos, target));
  * \endcode
  */
-inline CELER_FUNCTION real_type calc_miss_distance(Real3 const& pos,
-                                                   Real3 const& dir,
-                                                   real_type distance,
-                                                   Real3 const& target)
+inline CELER_FUNCTION real_type calc_miss_distance_sq(Real3 const& pos,
```

Comment:
Can we move the calculation inside `is_intercept_close` and remove this function as it is never used elsewhere.  Also, the function name itself is misleading as **miss_distance** stands for **sagitta** (at least in Geant4 and used in comments as different meaning elsewhere in celeritas codes)